packages/hooks/src/useP2PAdvertiserPaymentMethods.ts

```diff
@@ -0,0 +1,77 @@
+import { useCallback, useMemo } from 'react';
+import { useFetch, useInvalidateQuery, useRequest } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const type_to_icon_mapper = {
+    bank: 'IcCashierBankTransfer',
+    other: 'IcCashierOther',
+    ewallet: 'IcCashierEwallet',
+};
+
+/** A custom hook to fetch, create, update, and delete p2p advertiser payment methods */
+const useP2PAdvertiserPaymentMethods = (handleMutationError?: (error?: unknown) => void) => {
+    const invalidate = useInvalidateQuery();
+    const { client } = useStore();
+    const { is_authorize } = client;
+
+    /** Make a request to p2p_advertiser_payment_methods from API and onSuccess it will invalidate the cached data  */
+    const { mutate } = useRequest('p2p_advertiser_payment_methods', {
+        onSuccess: () => {
+            invalidate('p2p_advertiser_payment_methods');
+        },
```

Comment:
```suggestion
        onSuccess: () => invalidate('p2p_advertiser_payment_methods'),
```
@farhan-nurzi-deriv Nice nice 🚀